### DISCUSSION TITLE
Support for non OpenAI models

### DISCUSSION THREAD

**Discussion by gbrackenridge** [langflow-ai/langflow#121] [Ideas]
Apologies if this has been covered elsewhere on the boards - I didn't see anything. 

Any thoughts or pointers on supporting non OpenAI models? 

Have just started to peer through the code to see if I can redirect the calls to local LLM installs. Love to hear if anyone else if doing same or has some pointers.

I love the langflow setup, an amazing approach - well done!

**Comment by ngmisl** [comment]
you can use HuggingFaceHub and load the model remotely... but it's very slow

**Comment by carlosrcoelho** [comment]
Hey!

Thanks for sharing your question here. I'm sorry for not getting back to you sooner.
Langflow has gone through some major changes since then. I recommend trying out the new version for the best experience. If you have any issues, please open a new one; we'll be here to help.
